tests/bin/test_busd.py

```
@@ -1,169 +1,145 @@
-                "cwd": "work/T001",
-                "frame": "frames/impl/CLAUDE.md",
-                "goal": "test task",
-                "branch": "feat/T001"
-            }
-        }
+            Path(test_root, dir_path).mkdir(parents=True, exist_ok=True)
         
-        # mailboxに投函
-        mailbox_path = Path(self.test_root, 'mbox/bus/in/test.json')
-        mailbox_path.write_text(json.dumps(spawn_msg))
+        # busdモジュールをインポート（ここで初期化される）
```

Ruff (0.12.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)